frontend/public/components/notification-drawer.tsx

```diff
@@ -106,7 +107,9 @@ const getAlertNotificationEntries = (
           return (
             <NotificationEntry
               key={`${i}_${alert.activeAt}`}
-              description={getAlertDescription(alert) || getAlertMessage(alert)}
+              description={
+                <Linkify>{getAlertDescription(alert) || getAlertMessage(alert)}</Linkify>
```

Comment:
Use `LinkifyExternal` instead.

Reply:
@spadgett , so i played around with this, and i'm not sure its the behavior that we want. Maybe i'm wrong, but the link in the slack convo that created this pr is an internal link (image below). Are we expecting internal links here or external. While i agree if these links point us outside of the app we should use external, but otherwise, we have to re authenticate and open a new window. It seems more seamless not to use the external option. Thoughts on this?

![image](https://user-images.githubusercontent.com/35978579/90061379-11424080-dcb4-11ea-913f-53217c4ff814.png)

Reply:
@jcaianirh I'd expect these to almost always be external links. This one is the exception imo. We could have special handling if the link starts with the console, but that gets a bit involved. I'm not sure it's worth it.

Reply:
Maybe we can do something special for `UpdateAvailable` alerts.

Reply:
@spadgett ok, sounds good. I'll try that. Thanks for the feedback!